src/view/AlertStripeContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
Denne filen ble litt vanskelig å lesenår det er såpass mange conditonals for hva som blir returnert. 

Kanskje det blir lettere å lese med en switch case?
Eller ta ut hver predicate i en const, så gjøre if-else logikken?
Eventuelt visibleIfHoc de ulike AlertStripeAdvarsel komponentene og sende inn om de skal synnes som en prop ?

Reply:
Er nå endret til visibleIfHoc 😊